manubot/cite/tests/test_citekey.py

```diff
@@ -76,7 +76,7 @@
         pytest.param(
             "DOID:14330",
             dict(
-                standard_id="doid:14330",
+                standard_id="DOID:14330",
```

Comment:
There's one change in this PR I'd like to get some feedback on before merge @cthoyt and @agitter. In the past, all of our standardized citation keys used lowercase prefixes. However, this PR switches the prefix capitalization to the `preferred_prefix` in Bioregistry (for citation keys handled using `CURIE_Handler`).

Our past standard prefixes were all lowercases, as noted in https://github.com/manubot/manubot/pull/223:

> Prefixes are no longer case sensitive, although the standard form is lowercase.

But it's not clear this is a policy / requirement or just a statement that was true for all the prefixes with existing support.

The benefit of using `preferred_prefix` is that it will match the authoritative capitalization from the resource creator. For example, the Disease Ontology uses `DOID:14330` and not `doid:14330`. On the other hand, it adds complexity versus all lower prefixes and is perhaps less aesthetic (subjective).

I think the switch to preferred prefix makes sense, but wanted to get some additional feedback. Users are free to create citations with any capitalization of the prefix, but this is what the input citations get standardized to.

Reply:
I considered the same thing for the bioregistry. I believe case insensitivity and the ability to use synonyms is much more practical from the user's perspective, and then the bioregistry could be used to standardize it internally before rendering a citation.

Reply:
> I believe case insensitivity and the ability to use synonyms is much more practical from the user's perspective

FTR Manubot will (and does) support case insensitive prefixes in citation keys as inputs as well as synonymous prefixes. The question here is whether standard CURIEs should use preferred prefix capitalization or lowercase prefixes. Bioregistry currently uses lowercase, but that seemed to be a sticking point for some resource creators.

Reply:
oh then I would definitely suggest using the preferred prefix when available.

Reply:
I don't have a strong opinion on this but agree with using preferred prefix capitalization.